src/ontology/cl-edit.owl

```diff
@@ -31277,6 +31279,28 @@ AnnotationAssertion(rdfs:label obo:CL_4033037 "mucus secreting cell of tracheobr
 EquivalentClasses(obo:CL_4033037 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077)))
 SubClassOf(obo:CL_4033037 obo:CL_0002202)
 
+# Class: obo:CL_4033038 (lung resident memory CD4-positive, alpha-beta T cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24064670") Annotation(oboInOwl:hasDbXref "PMID:31265968") obo:IAO_0000115 obo:CL_4033038 "An alpha-beta CD4 T cell that resides in the lung. This cell expresses the resident marker CD69 and has the effector/memory phenotype: high CD44 (hyaluronic acid receptor) and low CD62L expression. A lung resident memory CD4-positive, alpha-beta T cell upregulates specific genes associated with tissue residency and downregulates genes associated with circulation and migration.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033038 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033038 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033038 "2023-04-27T10:15:45Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31265968") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4033038 "lung CD4-positive TRM")
+AnnotationAssertion(rdfs:label obo:CL_4033038 "lung resident memory CD4-positive, alpha-beta T cell")
+SubClassOf(obo:CL_4033038 obo:CL_0000897)
+SubClassOf(obo:CL_4033038 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002048))
+
+# Class: obo:CL_4033039 (lung resident memory CD8-positive, alpha-beta T cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28418771") Annotation(oboInOwl:hasDbXref "PMID:31265968") obo:IAO_0000115 obo:CL_4033039 "An alpha-beta CD8 T cell that resides in the lung. This cell expresses the resident marker CD69 and the chemokine receptor CXCR6. A lung resident memory CD8-positive, alpha-beta T cell upregulates specific genes associated with tissue residency and downregulates genes associated with circulation and migration.")
```

Comment:
The following appears in the associated issue, but appears missing in definition or comment here.

"High levels of CXCR6, integrin α1β1 (VLA-1: very late antigen-1) and αEβ7 (detected by CD49a and CD103, respectively) expression are unique hallmarks of this cell."

Reply:
The reason is that those hallmarks are specific for a subset of CD8 lung TRM that is CD103+, but not all CD8 lung TRMs. This term is supposed to be general, and not focus on a subset.

Reply:
The two sentences after the first may be applicable to all resident memory T cells, not specifically to lung-resident memory T cells.